lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -291,5 +296,24 @@ export type PerformanceEvent = {
      *
      * @type {?boolean}
      */
-    isNativeBroker?: boolean
+    isNativeBroker?: boolean,
+
+    /**
+     * The following are fields that may be emitted in native broker scenarios
```

Comment:
Suggest changing the names to hint that they are from mats.

Reply:
Done